### DISCUSSION TITLE
Sleep Mode and auto-brightness

### DISCUSSION THREAD

**Discussion by PricelessToolkit** [pixelit-project/PixelIt#82] [General]
I want to say a huge thank you for the project!!. I even made a short video about PixelIT
[https://youtu.be/Tpol4iNq4xo](https://youtu.be/Tpol4iNq4xo)
But in my opinion, there is an issue when I  turn off/on the light in the room, the display turns on and shows the date and time and remains turned on all night.  I use auto-brightness and sleep mode (OFF 24:00-14:00) (ON 14:00-24:00).
Please give me advice, where to look
Thanks

**Comment by o0shojo0o** [comment]
Hi @PricelessToolkit ,
you have found a bug with the auto-brightness function.
I will look at it as soon as possible and fix it! 

And thanks for the great video!

**Reply by o0shojo0o** [reply]
Ok, was found quickly is fixed in v0.3.13

**Reply by PricelessToolkit** [reply]
wow, that was fast !! thank you very much!  I have already updated it  I'll take a look today.
Thanks

**Reply by PricelessToolkit** [reply]
Hi, I tested the new firmware v0.3.13, everything now works as it should.
Thanks!
